libs/execution_context/ebpf_core.c

```diff
@@ -31,6 +31,24 @@ _ebpf_core_map_delete_element(ebpf_map_t* map, const uint8_t* key);
 
 #define EBPF_CORE_GLOBAL_HELPER_EXTENSION_VERSION 0
 
+static ebpf_helper_function_prototype_t _ebpf_map_helper_function_prototype[] = {
+    {1,
+     "ebpf_map_lookup_element",
+     EBPF_RETURN_TYPE_PTR_TO_MAP_VALUE_OR_NULL,
+     {EBPF_ARGUMENT_TYPE_PTR_TO_MAP, EBPF_ARGUMENT_TYPE_PTR_TO_MAP_KEY}},
+    {2,
```

Comment:
Would it work to replace the 2 here with 
```
   (uint32_t)ebpf_map_delete_element
```
which is defined in ebpf_helpers.h?   That would ensure the numbers are correct.

Reply:
compiler complains:
warning C4311: 'type cast': pointer truncation from 'ebpf_map_delete_element_t' to 'uint32_t'
       
      Note:
      `#define ebpf_map_delete_element ((ebpf_map_delete_element_t)3)`

Ideas?

Reply:
try `(uint32_t)(intptr_t)ebpf_map_delete_element`

Reply:
done.